db/items.sql

```diff
@@ -0,0 +1 @@
+CREATE TABLE items(id INTEGER PRIMARY KEY, name TEXT, category TEXT, image_name TEXT);
```

Comment:
You forgot to update the `category` column, right?
Also please include the `CREATE TABLE` statement for the `categories` table.

Reply:
I've added the table "category", and edited the table "items"